src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -0,0 +1,72 @@
+from snuba_sdk import Column, Function
+
+from sentry.sentry_metrics.utils import resolve_weak
+
+
+def __counter_sum_aggregation_on_session_status_factory(session_status, metric_ids, alias=None):
+    return Function(
+        "sumMergeIf",
```

Comment:
Out of curiosity: Why do we need the `Merge` modifier here? Doesn't snuba add that automatically? Or would that be `sumIfMerge` and thereby wrong?

Reply:
I am not sure actually but I have read in other places in the code that it was a `sumMergeIf`